cotolog.html

```diff
@@ -0,0 +1,290 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Глейси</title>
+  </head>
+  <body>
+    <header class="main-header">
+      <nav class="navigation">
+        <a href="index.html" class="logo-link"><img src="#" alt="Логотип Gllacy" class="logo"></a>
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <select class="select-cotolog">
+              <option disabled>Каталог</option>
+              <option value="Сливочное">Сливочное</option>
+              <option value="Щербеты">Щербеты</option>
+              <option value="Фруктовый лед">Фруктовый лед</option>
+              <option value="Мелорин">Мелорин</option>
+            </select>
+          </li>
```

Comment:
Вложенный список просто.

```suggestion
        <ul class="navigation-list">
          <li class="navigation-item">
            <a >Каталог</a>
            <ul>
              <li><a href="#">Сливочное</a></li>
              <li><a href="#">Щербеты</a></li>
              <li><a href="#">Фруктовый лед</a></li>
              <li><a href="#">Мелорин</a></li>
            </ul>
          </li>
```